Gemfile

```
@@ -85,4 +84,5 @@ group :test do
   gem 'simplecov', require: false
   gem 'webdrivers'
 end
+gem 'dotenv-rails', groups: %i[development test]
```

```suggestion
gem 'dotenv-rails'
```
it will be useful for all envs